Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/SubmissionIntroFragment.kt

```diff
@@ -1,32 +1,21 @@
 package de.rki.coronawarnapp.ui.submission
 
 import android.os.Bundle
-import android.view.LayoutInflater
 import android.view.View
-import android.view.ViewGroup
 import android.view.accessibility.AccessibilityEvent
 import androidx.fragment.app.Fragment
 import androidx.navigation.fragment.findNavController
+import de.rki.coronawarnapp.R
 import de.rki.coronawarnapp.databinding.FragmentSubmissionIntroBinding
 import de.rki.coronawarnapp.ui.doNavigate
-import de.rki.coronawarnapp.ui.viewLifecycle
+import de.rki.coronawarnapp.util.ui.viewBindingLazy
 
 /**
  * The [SubmissionIntroFragment] displays information about how the corona warning system works
  */
-class SubmissionIntroFragment : Fragment() {
+class SubmissionIntroFragment : Fragment(R.layout.fragment_submission_intro) {
 
-    private var binding: FragmentSubmissionIntroBinding by viewLifecycle()
-
-    override fun onCreateView(
-        inflater: LayoutInflater,
-        container: ViewGroup?,
-        savedInstanceState: Bundle?
-    ): View? {
-        // get the binding reference by inflating it with the current layout
-        binding = FragmentSubmissionIntroBinding.inflate(inflater)
-        return binding.root
-    }
+    private val binding: FragmentSubmissionIntroBinding by viewBindingLazy()
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
```

Comment:
`binding.lifecycleOwner = this` is missing (?)

Reply:
Only view layouts that use databinding, but this is confusing, let's make this automatic via `viewBindingLazy` as discussed, good idea!